src/commcare_cloud/fab/operations/staticfiles.py

```diff
@@ -32,7 +32,7 @@ def _version_static():
     cmd = 'resource_static'
     with cd(env.code_root):
         sudo(
-            'rm -f tmp.sh resource_versions.yaml; {venv}/bin/python manage.py {cmd}'.format(
```

Comment:
do you know what tmp.sh is from?

Reply:
I do not. I checked out back to 2016 (c4707daefaab03c8d7b4ab27b9c3172896f6f309) and didn't find any references to it.